account/build.gradle

```diff
@@ -17,6 +23,18 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	runtimeOnly 'mysql:mysql-connector-java'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	compile group: 'org.mockito', name: 'mockito-core', version: '3.6.28' // JUnit dependencies
```

Comment:
why's this needed as `compile ` and not `testCompile`?

Reply:
Changed the compile to testCompile